app/constants/preferences.ts

```diff
@@ -1,16 +1,26 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-const Preferences: Record<string, any> = {
-    CATEGORY_CHANNEL_OPEN_TIME: 'channel_open_time',
-    CATEGORY_CHANNEL_APPROXIMATE_VIEW_TIME: 'channel_approximate_view_time',
-    CATEGORY_DIRECT_CHANNEL_SHOW: 'direct_channel_show',
-    CATEGORY_GROUP_CHANNEL_SHOW: 'group_channel_show',
-    CATEGORY_EMOJI: 'emoji',
-    CATEGORY_SAVED_POST: 'flagged_post',
-    CATEGORY_FAVORITE_CHANNEL: 'favorite_channel',
-    CATEGORY_AUTO_RESET_MANUAL_STATUS: 'auto_reset_manual_status',
-    CATEGORY_NOTIFICATIONS: 'notifications',
+export const CATEGORIES_TO_KEEP: Record<string, string> = {
+    ADVANCED_SETTINGS: 'advanced_settings',
+    DIRECT_CHANNEL_SHOW: 'direct_channel_show',
+    GROUP_CHANNEL_SHOW: 'group_channel_show',
+    DISPLAY_SETTINGS: 'display_settings',
+    EMOJI: 'emoji',
+    NOTIFICATIONS: 'notifications',
+    SAVED_POST: 'flagged_post',
+    SIDEBAR_SETTINGS: 'sidebar_settings',
+    TEAMS_ORDER: 'teams_order',
+    THEME: 'theme',
+};
+
+const CATEGORIES: Record<string, string> = {
+    ...CATEGORIES_TO_KEEP,
+    FAVORITE_CHANNEL: 'favorite_channel',
```

Comment:
Why favorite channel is not on the categories to keep?

Reply:
cause is no longer being used.. categories now hold the favorites `category` on the sidebar. confusing I know